app/models/service/aggregation.rb

```diff
@@ -149,7 +149,7 @@ def aggregate_hardware_arel(virtual_column_name, aggregation_sql, options = {})
     def aggregation_where_clause(arel, subtree_services)
       arel.grouping(
         subtree_services[:id].eq(arel[:id])
-         .or(subtree_services[:ancestry].matches(ancestry_ilike))
+         .or(subtree_services[:ancestry].matches(ancestry_ilike, nil, true))
```

Comment:
Also, Keenan [clearly knows about this](https://github.com/rails/arel/pull/403), but for those who might not bother dealing with AREL internals, this is the proper way to flag a case sensitive (`LIKE`) query in postgres using AREL:

- https://github.com/rails/arel/blob/v7.1.4/lib/arel/nodes/matches.rb#L7
- https://github.com/rails/arel/blob/v7.1.4/lib/arel/visitors/postgresql.rb#L11